### PR TITLE
Update countries concept

### PR DESCRIPTION
**Pull request by danil-topchiy**
#10

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -189,18 +189,22 @@ def test_new_it_guy(edap):
...
+    assert not edap.subobject_exists_at(f"cn={country_cname},{edap.COUNTRIES}", "posixGroup")
+    edap.create_country(country_cname)
+    assert edap.subobject_exists_at(f"cn={country_cname},{edap.COUNTRIES}", "posixGroup")
+    assert edap.object_exists(f"&(commonName={country_cname})(description={country_description})", "posixGroup")
+    assert edap.get_country(country_cname)['cn'][0] == country_cname.encode('utf-8')
+    assert edap.get_country(country_cname)['description'][0] == country_description.encode('utf-8')
+
+
+def test_label_country(edap):
+    assert edap.label_country("cz") == "Czech Republic"
```
Comment (matejak) [inline test.py]: This needs updating, Czechia != Czech Republic.

File: test.py [hunk 2]
```diff
@@ -189,18 +189,22 @@ def test_new_it_guy(edap):
...
+    assert edap.label_country("cz_prg") == "Czech Republic"
```
Comment (matejak) [inline test.py]: `cz_prg` is not a country code. I think that it should pass, so the label function should allow a franchise code in form of `<country_code>` or `<country_code>_<something>`, i.e. `cz` and `cz_prg` are OK, but `cz_` is not OK. I guess that you can call `'cz_prg'.split('_', 1)`, and the first split element has to be the ISO code, while the second split element has to be non-empty if it exists.

File: src/edap/constants.py [hunk 3]
```diff
@@ -1,20 +1,253 @@
...
+    "co": "Colombia",
+    "km": "Comoros",
+    "cg": "Congo - Brazzaville",
+    "ck": "Cook Islands",
+    "cr": "Costa Rica",
+    "hr": "Croatia",
+    "cu": "Cuba",
+    "cw": "Curaçao",
+    "cy": "Cyprus",
+    "cz": "Czechia",
```
Comment (matejak) [inline src/edap/constants.py:64]: Nice one, we are indeed officially Czechia for some time now :-)

**Review by matejak — CHANGES_REQUESTED** [review]
Tests won't pass, so some fixing is required.

More importantly, I apologize for confusion caused by the "Country" concept name - the original "franchise" was OK, as it allows more flexibility than a country (i.e. an enterprisey organization may have more franchises in one country if needed). Therefore, please change `country` to `franchise` (except `COUNTRIES_CODES` that are indeed country-related codes).

Otherwise, there are no merge blockers thank you for the PR!

File: src/edap/constants.py (hunk 3 above)
Comment (danil-topchiy) [reply src/edap/constants.py:64]: Even I was surprised when saw at first, although I haven't ever lived in Czechia :)

File: test.py (hunk 2 above)
Comment (danil-topchiy) [reply test.py]: I did `splitted_code = code.split('_')` and checking if `splitted_code[1]` exists, to reject `cz___` as well.

File: test.py (hunk 1 above)
Comment (danil-topchiy) [reply test.py]: done

**Comment by danil-topchiy** [comment]
Changed all names back to franchise and made required fixes

**Comment by matejak** [comment]
Could you please rebase so we can check out the CI?

**Comment by danil-topchiy** [comment]
@matejak rebased it, although don't have permissions to enable EnterpriseyIntranet/edap pipelines on my travis account.

**Review by matejak — APPROVED** [review]

**Comment by matejak** [comment]
I have forgot that I have to enable tests on Travis first. Anyway, good job with those fixes and actually with the whole PR!
I have a couple of remarks beyond this PR:

1. Please add a test for the functionality that rejects some franchise names in another PR - it does some non-trivial things. (I didn't want to keep this PR waiting).
2. When addressing the review, it will help me if you don't force-push, but if you just pile commits up. That way, it is easier to see what got changed. I can always squash merge, so the history is nice, or you can perform an interactive rebase just before the merge :-)

**Comment by danil-topchiy** [comment]
@matejak thanks for review! 
> Please add a test for the functionality that rejects some franchise names in another PR - it does some non-trivial things. (I didn't want to keep this PR waiting).

Added it in new PR.
>When addressing the review, it will help me if you don't force-push, but if you just pile commits up. That way, it is easier to see what got changed. I can always squash merge, so the history is nice, or you can perform an interactive rebase just before the merge :-)

Ok, no problem!

**Pull request merged**